python/feldera/sql_context.py

```diff
@@ -366,6 +380,86 @@ def foreach_chunk(self, view_name: str, callback: Callable[[pandas.DataFrame, in
         handler = CallbackRunner(self.client, self.pipeline_name, view_name, callback, queue)
         handler.start()
 
+    def connect_source_kafka(
+        self,
+        table_name: str,
+        connector_name: str,
+        config: dict,
+        fmt: JSONFormat | CSVFormat
+    ):
+        """
+        Tells Feldera to read the data from the specified kafka topic.
+
+        :param table_name: The name of the table.
+        :param connector_name: The unique name for this connector.
```

Comment:
Why do connectors need names?

Reply:
So many names to keep track of!
I think that the default should be the table/view name. It most cases there will be exactly 1 connector per table/view.

Reply:
Yeah, I am unsure what to do for this. 
Names are also required for delta table connectors. Auto generating a default name might be a good idea.

Reply:
For this API, it might make sense to auto-generate connector names by default as Mihai said.